src/integrationTest/groovy/wooga/gradle/unity/tasks/AddUPMPackagesTaskIntegrationSpec.groovy

```diff
@@ -6,33 +6,48 @@ import com.wooga.spock.extensions.uvm.UnityInstallation
 import groovy.json.JsonOutput
 import groovy.json.JsonSlurper
 import net.wooga.uvm.Installation
+import org.gradle.api.file.Directory
 import spock.lang.Requires
+import wooga.gradle.unity.UnityIntegrationSpec
 import wooga.gradle.unity.UnityTaskIntegrationSpec
 
-class AddUPMPackagesTaskIntegrationSpec extends UnityTaskIntegrationSpec<AddUPMPackages> {
-
+class AddUPMPackagesTaskIntegrationSpec extends UnityTaskIntegrationSpec {
```

Comment:
Why are you no longer using the Generic setup for this task type?

Reply:
See above; as these tasks that modify the project manifest no longer invoke unity themselves, they are not `UnityTask`s.